### DISCUSSION TITLE
Kafka Stream Dependencies

### DISCUSSION THREAD

**Discussion by jkowall** [jaegertracing/jaeger#3006] [Show and tell]
Hello folks, I wanted to post a question for the community. I know we have the spark-dependencies option to generate the dependencies graphs and views. The team here at Logz has been running Kafka Streams in production to do the same thing on our side. We wanted to contribute (and of course maintain) the code but figured it should be in the jaegertracing repo. Any comments or feedback on how we should proceed?

We have the code, code, and such ready to go.

**Comment by yurishkuro** [comment]
I would just include the code in https://github.com/jaegertracing/jaeger-analytics-java. Ideally the business logic should be written independently of the streaming framework, e.g. "given a trace produce a DAG". This way the same module can be reused with different streaming systems.

**Comment by albertteoh** [comment]
Issue created: jaegertracing/jaeger-analytics-java#63
